Cython/Compiler/ExprNodes.py

```diff
@@ -9436,6 +9436,12 @@ def generate_cyfunction_code(self, code):
         if def_node.local_scope.parent_scope.is_c_class_scope and not def_node.entry.is_anonymous:
             flags.append('__Pyx_CYFUNCTION_CCLASS')
 
+        # mutually exclusive as far as "inspect" is concerned
+        if getattr(def_node, "is_asyncgen", None):
+            flags.append('__Pyx_CYFUNCTION_ASYNCGEN')
```

Comment:
Is this used anywhere?

Reply:
Not really, it's leftover from previous attempts. Although it's mutually exclusive with the other one... (see inspect.isasyncgenfunction py3.6) I can see people forgetting to add it, if the protocol is finally defined and implemented. I can remove it, if you think it's unnecessary.

Reply:
These are Cython-internal flags, though, not known to CPython.